lib/addressfinder/batch_verification.rb

```diff
@@ -0,0 +1,68 @@
+module AddressFinder
+  class BatchVerification
```

Comment:
Looks like you could use class inheritance to avoid having to copy paste the whole boilerplate work that comes from concurrency?

Reply:
When I add the PV batch capability, I'll extract the common functionality